docs/4.1/utilities/icons.md

```diff
@@ -517,6 +517,14 @@ All icons that are currently available for use
         <code>.icon-social-media-placeholder</code>
       </td>
     </tr>
+    <tr>
+      <td>
+        <p class="icon-menu-icon"></p>
```

Comment:
this reads really weird, i would think about omitting the 'icon' from the image name

Reply:
I don't get it. This is in the entire document. 😃

Reply:
no, i mean that the word 'icon' should not be part of the file name, it should still be available as a prefix. so e.g. .icon-menu-burger with a file 063-menu-burger.svg ;)